applepay/payment.py

```diff
@@ -62,6 +64,26 @@ def _generate_symmetric_key(self, shared_secred):
 
         return sha.digest()
 
+    @staticmethod
+    def signing_time_is_valid(signature, current_time, threshold):
+        """ Given a detached top-level CMS signature, validate the 'signingTime'
+        attribute against the current time and a time-delta threshold.
+
+        If the difference between the current time and the 'signingTime' exceeds
+        the threshold, the token should be considered invalid.
+
+        :param signature: Base64 encoded detached CMS signature data.
+        :type: str
+        :param current_time: Current system time to compare the token against.
+        :type: offset-aware datetime
+        :param threshold: Amount of time to consider the token valid.
+        :type: timedelta
+        :return: False if the signing time exceeds the threshold, otherwise True
+        :rtype: bool
+        """
+        signing_time = payment_utils.retrieve_signature_signing_time(signature)
```

Comment:
I suppose that there is exception handling in `payment_utils.retrieve_signature_signing_time()` that makes a try block here unnecessary?